tests/iscsi/iscsi_server.pm

```diff
@@ -16,91 +16,249 @@ use strict;
 use warnings;
 use testapi;
 use mm_network;
+use mm_tests;
 use lockapi;
 use version_utils qw(is_sle is_leap);
 use mmapi;
-use utils 'zypper_call';
+use utils qw(zypper_call systemctl type_string_slow_extended);
 use yast2_widget_utils 'change_service_configuration';
 use x11utils 'turn_off_gnome_screensaver';
-use y2_module_consoletest;
```

Comment:
I'm not sure why we have just imported that module, but in general, why don't we inherit from it, as it was designed exactly for the tests like this one.

Reply:
Alright, I will rewrite the test suite to use ``lib/y2_module_guitest.pm``.